lib/uffizzi/cli/preview.rb

```diff
@@ -179,10 +205,9 @@ def check_activity_items_state(activity_items, containers_spinners)
     def handle_delete_command(deployment, project_slug)
       return Uffizzi.ui.say("Preview should be specified in 'deployment-PREVIEW_ID' format") unless deployment_name_valid?(deployment)
 
-      hostname = ConfigFile.read_option(:hostname)
       deployment_id = deployment.split('-').last
```

Comment:
I see this logic in a few places. Can we organize the logic as follows:

```
deployment_id = read_deployment_id(deployment_name)
return Uffizzi.ui.say("Preview should be specified in 'deployment-PREVIEW_ID' format") if deployment_id.nil?
```

So in this case we don't need `deployment_name_valid?` method at all